### DISCUSSION TITLE
Material you

### DISCUSSION THREAD

**Discussion by Maha-Rajan** [iSoron/uhabits#1506] [Feature Requests]
Please consider improving the app to the latest m3 design language

**Comment by OneRoomer** [comment]
Maybe in addition to you post, this is what it looks right now using a Pixel 7 Pro

![Screenshot_20230704-081007](https://github.com/iSoron/uhabits/assets/107799871/83566939-2343-4c1e-b89f-4aa7578cb465)

**Reply by alr86** [reply]
For now, you can use "Arcticons" or "Iconeration" apps
